babel.config.json

```diff
@@ -237,6 +237,7 @@
 
           "@department-of-veterans-affairs/platform-forms-system/exports": "./src/platform/forms-system/exportsFile.js",
           "@department-of-veterans-affairs/platform-forms-system/AddressViewField": "./src/platform/forms-system/src/js/components/AddressViewField.jsx",
+          "@department-of-veterans-affairs/platform-forms-system/VaMemorableDateField": "./src/platform/forms-system/src/js/web-component-fields/VaMemorableDateField",
```

Comment:
None of the web components are in this list yet. Not sure why? But it seems like the right step here.